moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -453,7 +453,7 @@ class PlanningSceneMonitor : private boost::noncopyable
 
   typedef std::map<const robot_model::LinkModel*, std::vector<std::pair<occupancy_map_monitor::ShapeHandle, std::size_t> > > LinkShapeHandles;
   typedef std::map<const robot_state::AttachedBody*, std::vector<std::pair<occupancy_map_monitor::ShapeHandle, std::size_t> > > AttachedBodyShapeHandles;
-  typedef std::map<std::string, std::vector<std::pair<occupancy_map_monitor::ShapeHandle, const Eigen::Affine3d*> > > CollisionBodyShapeHandles;
+  typedef std::map<std::string, std::vector<std::pair<occupancy_map_monitor::ShapeHandle, Eigen::Affine3d*> > > CollisionBodyShapeHandles;
```

Comment:
This is a protected typedef. If it had been private I wouldn't have reverted it (after confirming that the member variable declared with the typedef also doesn't leave private scope), as the const version should work fine with non-const objects. But since it is protected it is conceivable that there are subclasses out there somewhere using the typedef that would break, so I figured better safe than sorry.